Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -84,6 +84,20 @@ AVFoundationEnabled:
       "PLATFORM(WATCHOS)": false
       default: true
 
+AbortSignalAnyOperationEnabled:
+  type: bool
+  category: dom
+  status: preview
+  humanReadableName: "AbortSignal.any() operation"
+  humanReadableDescription: "AbortSignal.any() operation"
```

Comment:
The description is perhaps a bit terse...

Reply:
It seemed clear enough to me. I don't think I would be able to explain what the operation does in a short enough sentence. Do you have a better proposal?

Reply:
That's how the other features are defined, even though it doesn't provide very much more context -- at least it explains that the purpose of the toggle is to Enable the feature.

Reply:
I added "Enable"